src/inverseRobot.js

```diff
@@ -25,6 +25,16 @@
  */
 function inverseRobot(robot) {
   // write code here
+  const repairedRobot = {};
+
+  for (const key in robot) {
+    if (repairedRobot[robot[key]]) {
```

Comment:
пункт 2 чекліста, так краще не перевіряти присутність `key` в об'єкта.
```suggestion
    if (repairedRobot.hasOwnProperty(robot[key])) {
```